bears/c_languages/CPPLintBear.py

```diff
@@ -26,12 +32,20 @@ class CPPLintBear:
     def create_arguments(filename, file, config_file,
                          max_line_length: int=79,
                          cpplint_ignore: typed_list(str)=(),
-                         cpplint_include: typed_list(str)=()):
+                         cpplint_include: typed_list(str)=(),
+                         indent_size: int=2,
+                         use_spaces: bool=True,):
         """
         :param max_line_length: Maximum number of characters for a line.
         :param cpplint_ignore:  List of checkers to ignore.
         :param cpplint_include: List of checkers to explicitly enable.
+        :param use_spaces:      Only `True` value is acceptable.
+        :param indent_size:     Acceptable indentation size is 2.
```

Comment:
This isnt strong enough. For consistency, and to ensure the reader is convinced, it should be "Only an indent size of 2 is permitted by this bear."

(fwiw, I have also created https://github.com/coala/coala-bears/issues/1192 to add the same docstrings to CheckstyleBear)